packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -215,7 +215,7 @@ module.exports = ctx => ({
         items: [
           { text: 'Pricing', link: 'https://www.okta.com/pricing/#customer-identity-products' },
           { text: 'Integrate with Okta', link: '/okta-integration-network/' },
-          { text: 'Change log', link: '/docs/release-notes/' },
+          { text: 'Release Notes', link: '/docs/release-notes/' },
```

Comment:
I'm sure this is fine, but can you run this update by Chelsea, Kim, and Janet? In the past, I remember they really wanted to cling onto the API "change log" terminology.

Reply:
Slack signoff by Janet.